packages/open-next/src/build/helper.ts

```diff
@@ -167,47 +167,56 @@ export async function esbuildAsync(
   }
 }
 
+/**
+ * Recursively delete files.
+ *
+ * @see `traverseFiles`.
+ *
+ * @param root Root directory to search.
+ * @param conditionFn Predicate used to delete the files.
+ */
 export function removeFiles(
   root: string,
-  conditionFn: (file: string) => boolean,
-  searchingDir: string = "",
+  conditionFn: (paths: {
```

Comment:
Maybe we could extract the paths as it's own type since it is used here and in the 2 traverseFiles